sources/lib/Resources/config/services/pomm.yml

```diff
@@ -8,10 +8,16 @@ services:
         arguments: ['@pomm']
         tags:
             - { name: request.param_converter, converter: pomm.entity }
+
     pomm.serializer.normalizer:
         class: 'PommProject\SymfonyBridge\Serializer\Normalizer\FlexibleEntityNormalizer'
         tags:
-            - { name: serializer.normalizer }
+            - { name: serializer.normalizer, priority: 999 }
```

Comment:
Just use priority 0 or 1 then :p. Also it looks like priority should be between `-1` and `255`.